src/components/Clock/Clock.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export class Clock extends React.Component {
+  state = {
+    date: new Date(),
+  };
+
+  componentDidMount() {
+    this.timer = setInterval(() => {
+      this.setState({
+        date: new Date(),
```

Comment:
Можно сразу тут прописать
```suggestion
        date: new Date().toLocaleTimeString()
```
чтоб потом везде не дописывать этот метод, а использовать просто date